hydra-node/src/Hydra/Network/Ouroboros.hs

```diff
@@ -236,7 +235,7 @@ withOuroborosNetwork tracer localHost remoteHosts networkCallback between = do
    where
     initiator =
       MuxPeer
-        (contramap (WithHost localHost . TraceSendRecv) tracer)
```

Comment:
My reasoning has been that this information is mostly useless operationally: 
1. I assume we trust the underlying network implementation to do its job so there's no need for such signals
2. We know it can detect failed or failing connections, so failure to send messages will be reported
3. It could be useful to know what's exactly sent or receive but serialisation tests already ensure that
4. What's important to report for hydra node is the consequence (or lack thereof) of such low-level messages
5. Should we need to, we can always tcpdump this

Overall, I thought the signal-to-noise ratio was low.